005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -1 +1,197 @@
-\chapter{Team 3 Agent Design}\label{team_3_agent_design}
\ No newline at end of file
+\chapter{Team 3 Agent Design}\label{team_3_agent_design}
+
+\section{The Agent}\label{sec:the_agent}
+Team 3’s agents design is based on the Theory of Reasoned Action (TRA) and its extension, the Theory of Planned Behaviour (TPB). The agents are designed to have a shiftable personality dependent on the three variables in Figure \ref{fig:TRA-diagram} that TPB  \cite{AJZEN1991179} links to the behaviours of an agent. These three variables, together with a social network, define its personality as a whole, \cite{victor} and influence the actions it will take.
```

Comment:
```suggestion
Team 3’s agent design is based on the Theory of Reasoned Action (TRA) and its extension, the Theory of Planned Behaviour (TPB). The agents are designed to have a shiftable personality dependent on the three variables in Figure \ref{fig:TRA-diagram} that TPB  \cite{AJZEN1991179} links to the behaviours of an agent. These three variables, together with a social network, define its personality as a whole, \cite{victor} and influence the actions it will take.
```